swapchain.cpp

```diff
@@ -224,6 +224,10 @@ VKAPI_ATTR void VKAPI_CALL vkSetSwapchainCallback(VkSwapchainKHR swapchain,
                                                   void callback(void*, uint8_t*,
                                                                 size_t),
                                                   void* user_data) {
+  // RenderDoc Wraps the swapchain so need to get the original one here.
```

Comment:
Can we make this more generic than RENDERDOC. This is really for unwrapping from any higher layer.